README.md

```diff
@@ -40,6 +40,7 @@ Start with the [Hello World](basics/hello-world) and then jump to the topic you
 - [Jupyter Widgets](widget-tracker/jupyter-widgets)
 - [Kernel Output](advanced/kernel-output)
 - [Kernel Messaging](advanced/kernel-messaging)
+- [Server Hello World](advanced/myserverextension)
```

Comment:
Maybe we could name it `server-extension` to follow the same style as for the other examples?

Reply:
Yes indeed thanks for the suggestion